adafruit_ssd1306/__init__.py

```diff
@@ -1 +1,2 @@
-from adafruit_ssd1306.ssd1306 import *
+"""force classes to be loaded"""
+from adafruit_ssd1306.ssd1306 import SSD1306_SPI, SSD1306_I2C
```

Comment:
This was a weird pattern done in some of the drivers. Instead, rename `ssd1306.py` to `adafruit_ssd1306.py` in the top level directory.

Try renaming the base class to _SSD1306 to make it private to the module. I hadn't thought of doing that before.

Reply:
moved and renamed ssd1306.py.  changed the name of  SSD1306 to _SSD1306